packages/odyssey-react-mui/src/Tile.tsx

```diff
@@ -50,23 +50,23 @@ export type TileProps = {
 ) &
   HtmlProps;
 
-const ImageContainer = styled.div<{
-  odysseyDesignTokens: DesignTokens;
-  hasMenuItems: boolean;
-}>`
-  display: flex;
-  align-items: flex-start;
-  max-height: 64px;
-  margin-block-end: ${(props) => props.odysseyDesignTokens.Spacing5};
-  padding-right: ${(props) =>
-    props.hasMenuItems ? props.odysseyDesignTokens.Spacing5 : 0};
-`;
+// const ImageContainer = styled.div<{
+//   odysseyDesignTokens: DesignTokens;
+//   hasMenuItems: boolean;
+// }>`
+//   display: flex;
+//   align-items: flex-start;
+//   max-height: 64px;
+//   margin-block-end: ${(props) => props.odysseyDesignTokens.Spacing5};
+//   padding-right: ${(props) =>
+//     props.hasMenuItems ? props.odysseyDesignTokens.Spacing5 : 0};
+// `;
 
-const MenuButtonContainer = styled.div<{ odysseyDesignTokens: DesignTokens }>`
-  position: absolute;
-  right: ${(props) => props.odysseyDesignTokens.Spacing3};
-  top: ${(props) => props.odysseyDesignTokens.Spacing3};
-`;
+// const MenuButtonContainer = styled.div<{ odysseyDesignTokens: DesignTokens }>`
+//   position: absolute;
+//   right: ${(props) => props.odysseyDesignTokens.Spacing3};
+//   top: ${(props) => props.odysseyDesignTokens.Spacing3};
+// `;
```

Comment:
Why'd we comment these out?

Reply:
From an earlier change that I forgot to undo... will revert this as part of the cleanup